docs/src/stories/DefaultValues/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 import DropdownTreeSelect from '../../../../src'
-import '../../../../dist/styles.css'
+// import '../../../../dist/styles.css'
```

Comment:
Did you run `yarn build`? This file gets created by the build and this import tells Webpack to bundle it for the docs.

Reply:
Ow! 
Here's the thing - I cloned the repo, installed the modules and then (as per the README.md) immediately did a `npm run demo`. For some reason, that styles.css was not generated, hence console would throw an invalid import error. 
It makes full sense now & the css file gets created as expected. I've thus reverted the commit which commented out the line.
Thanks.

Reply:
`npm run demo` is for local development. It doesn't use the filesystem, rather uses the in-memory artifacts.